index.js

```diff
@@ -34,7 +34,10 @@ module.exports = function (linkHeader) {
     .map(parseLink)
     .filter(function (x) { return x && x.rel; })
     .reduce(function (acc, x) {
-      acc[x.rel] = x;
+      x.rel.split(/\s+/).forEach(function (rel) {
```

Comment:
totally agree, sorry was trying to fit surrounding style.

it looks like you prefer the function definitions at the top, then?